sql/branch.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+    ALTER TABLE participants DROP COLUMN goal;
```

Comment:
@whit537 - Are you sure we want to start dropping columns right now? Wouldn't it make sense to leave it all in there, and delete after we're done migrating?